src/wmtk/SimplicialComplex.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include "simplex/SimplexCollection.hpp"
 #include "simplex/closed_star.hpp"
-#include "simplex/coface_cells.hpp"
+#include "simplex/top_level_cofaces.hpp"
```

Comment:
Why did you replace `coface_cells` by `top_level_cofaces`?

Reply:
i didn't find the name very descriptive/direct. right now the name is "the cells that are cofaces". instead the name now implies "the cofaces that are at the top level". the specialization of which cofaces we are looking at seems more apt than the specialization of which cells we are looking at (i've had trouble remembering this definition at least?)

Reply:
It's fine for me. I understand both.